### DISCUSSION TITLE
[Feature] Video Deinterlace Support

### DISCUSSION THREAD

**Discussion by jamesharding26** [immich-app/immich#12290] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Many of the home movies I am uploading are from the 2000s and are therefore interlaced. When Immich encodes them, it does not de-interlace them properly for display. Would it be possible to detect interlacing and de-interlace the file accordingly?

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#10562
